README.md

```diff
@@ -158,14 +158,19 @@ Please see [Management Center Repository](https://github.com/hazelcast/managemen
 
 # Hazelcast Kubernetes
 
-Hazelcast maintains two Docker Images for Kubernetes Environment. Below are the links to their README files. You can also find Docker Images on [Docker Hub](https://store.docker.com/profiles/hazelcast) page.
+Hazelcast is prepared to work in the Kubernetes environment. For details, please check:
 
-* [hazelcast-kubernetes](https://github.com/hazelcast/hazelcast-docker/blob/master/hazelcast-kubernetes/README.md)
-* [hazelcast-enterprise-kubernetes](https://github.com/hazelcast/hazelcast-docker/blob/master/hazelcast-enterprise-kubernetes/README.md)
+* Hazelcast Helm Charts:
```

Comment:
Are we gonna mention the one hosted by kubernetes chart repository?

Reply:
Good point. Added.